tests/test_complex.py

```diff
@@ -175,3 +175,8 @@ def test_slice_to_end_infinite_lists():
 def test_interleave():
     stack = run_vyxal("⁽›1Ḟ ⁽⇧1Ḟ Y")
     assert stack[-1][:6] == [1, 1, 2, 3, 3, 5]
+
+
+def test_compressed_strings():
```

Comment:
This isn't really a unit test. It's fine to keep, but there should probably also be a direct test for the functionality of `to_base_digits`

Reply:
wdym? There is and definitely always has been a test for that function